src/Mutation/PostObjectUpdate.php

```diff
@@ -95,6 +95,14 @@ public static function mutate_and_get_payload( $post_type_object, $mutation_name
 				throw new UserError( sprintf( __( 'Sorry, you are not allowed to update a %1$s', 'wp-graphql' ), $post_type_object->graphql_single_name ) );
 			}
 
+			/**
+			 * If the existing post was authored by another author, ensure the requesting user has permission to edit it
+			 */
+			if ( $existing_post->post_author !== get_current_user_id() && ! current_user_can( $post_type_object->cap->edit_others_posts ) ) {
+				// translators: the $post_type_object->graphql_single_name placeholder is the name of the object being mutated
```

Comment:
Does this comment get removed at some point or is this just information for translations that will stay in the codebase?

Reply:
It stays in. Ya, it's used by tooling for translating WordPress plugins so translators know what variables in strings are as they work on translating strings.